webodf/lib/ops/OpApplyHyperlink.js

```diff
@@ -101,7 +101,8 @@ ops.OpApplyHyperlink = function OpApplyHyperlink() {
             linkNode = createHyperlink(ownerDocument, hyperlink);
             node.parentNode.insertBefore(linkNode, node);
             linkNode.appendChild(node);
-            if (modifiedParagraphs.indexOf(paragraph) === -1) {
+            if (paragraph !== null
```

Comment:
I'm a smidge torn here, as a null paragraph would be very unexpected at this point. Would maybe be better to just hard-case the return type of the ``odfUtils.getParagraphElement`` call above to ``!Element``?